evm/db/__init__.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from evm.utils.module_loading import (
+    import_string,
+)
+
+
+DEFAULT_DB_BACKEND = 'evm.db.backends.memory.MemoryDB'
+
+
+def get_db_backend_class(import_path=None):
+    if import_path is None:
+        import_path = os.environ.get(
+            'EVM_DB_BACKEND_CLASS',
+            DEFAULT_DB_BACKEND,
+        )
+    return import_string(import_path)
+
+
+def get_db_backend():
```

Comment:
I've typically allowed passing in the `import_path` into this function as well so that users can customize the backend with *either* the environment variable, or by directly passing through the backend class import path they want to use.